subprojects/common/time/src/main/kotlin/com/avito/time/DefaultTimeProvider.kt

```diff
@@ -12,20 +10,12 @@ import java.util.concurrent.TimeUnit
 
 /**
  * Used in android runtime < 26, so no java.time API yet
- * consider using ThreeTenABP
+ * consider using https://developer.android.com/studio/write/java8-support#library-desugaring
  */
-class DefaultTimeProvider(loggerFactory: LoggerFactory) : TimeProvider {
-
-    private val logger = loggerFactory.create<DefaultTimeProvider>()
+class DefaultTimeProvider : TimeProvider {
 
     private val locale: Locale = Locale.getDefault()
 
-    override val timeZone: TimeZone = TimeZone.getDefault()
-
-    init {
-        logger.debug("Creating new time provider: Locale = $locale; TimeZone = $timeZone")
-    }
-
     override fun isSameDay(date1: Date, date2: Date): Boolean {
```

Comment:
Another example. It seems like an extension function for Date:

```kotlin
date1.isSameDay(data2)
```

I don't see reasons to have this function in a big interface an pass it to all usages.

Reply:
fixed